cob_default_robot_config/cob3-2/arm_joint_configurations.yaml

```diff
@@ -1,6 +1,6 @@
 joint_names: ["arm_1_joint","arm_2_joint","arm_3_joint","arm_4_joint","arm_5_joint","arm_6_joint","arm_7_joint"]
```

Comment:
What are the joints called after when you spawn them in a robot_id namespace?
Do they get a prefix similar to the tf_frames?